cider/exec/template/InValuesIR.cpp

```diff
@@ -126,6 +129,59 @@ std::unique_ptr<CodegenColValues> CodeGenerator::codegenInValues(
   return std::make_unique<FixedSizeColValues>(result, lhs_null);
 }
 
+std::unique_ptr<CodegenColValues> CodeGenerator::codegenInValuesString(
+    const Analyzer::InValues* expr,
+    const CompilationOptions& co) {
```

Comment:
Here we write a specific `codegen` method for `String` as `TwoValueColValues` always need separate logic and makes some duplicate code. Not sure is there a way to provide other kind of abstract to avoid this.